src/management-system-v2/app/(dashboard)/[environmentId]/profile/user-profile.tsx

```diff
@@ -64,6 +91,31 @@ const UserProfile: FC<{ userData: User }> = ({ userData }) => {
         }}
       />
 
+      <Modal
+        title="Change your email"
+        open={changeEmailModalOpen}
+        closeIcon={null}
+        onCancel={() => setChangeEmailModalOpen(false)}
+        onOk={changeEmailForm.submit}
+        destroyOnClose
+      >
+        <Alert
+          type="warning"
+          message="We'll send a sign in link to your new email, if you don't open it in this browser your email won't be changed"
```

Comment:
The message is actually false, it was from previous code, where that was needed, but that's not the case anymore, you don't have to do it in the same browser. You just have to be signed in, which I think is necessary, because if you enter a wrong email then somebody could take over your account.